azure/packages/azure-client/src/AzureAudience.ts

```diff
@@ -14,6 +14,7 @@ import { type AzureMember, type AzureUser, type IAzureAudience } from "./interfa
  * @remarks Operates in terms of {@link AzureMember}s.
  *
  * @public
+ * @deprecated This class will no longer be exposed. use {@link IAzureAudience} instead
```

Comment:
Looks like this class is on track to be entirely removed, not just hidden, right?
```suggestion
 * @deprecated This class will be removed. use {@link IAzureAudience} instead
```